packages/web/app/src/router.tsx

```diff
@@ -343,28 +364,46 @@ const organizationRoute = createRoute({
   errorComponent: ErrorComponent,
 });
 
-const organizationIndexRoute = createRoute({
+export const organizationLayoutRoute = createRoute({
```

Comment:
This is a new pathless layout route. The component is the same component which we currently use to wrap org pages. Org pages call this layout route their parent.